packages/flutter_splash_tizen/example/pubspec.yaml

```diff
@@ -0,0 +1,28 @@
+name: flutter_splash_tizen_example
+description: Demonstrates how to use the flutter_splash_tizen plugin.
+
+publish_to: 'none'
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+  flutter_splash_tizen:
+    path: ../
+
+  cupertino_icons: ^1.0.2
```

Comment:
Sort the dependencies alphabetically and remove extra newlines between the dependencies.